stargz/testutil.go

```diff
@@ -0,0 +1,41 @@
+// Copyright 2019 The Go Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package stargz
+
+// Makes minimal Reader of "reg" and "chunk" without tar-related information.
```

Comment:
Document this the normal Go way:
https://github.com/golang/go/wiki/CodeReviewComments#comment-sentences

But also, let's rename it to not include "Stub".

But why is it exported at all? Can't you just put this in the test package where it's needed? Then naming & docs aren't quite as important. But once it's public API, the quality bar is much higher.

Reply:
Thank you for your review. I came up with another implementation not to export the function and fixed it.